src/controller.cpp

```diff
@@ -290,8 +290,7 @@ void Controller::serverUnavailable() {
 
   m_nextStep = ServerUnavailable;
 
-  if (m_state == StateOn || m_state == StateSwitching ||
```

Comment:
What is the new user experience after this change? I think the user will need to go to the app, toggle the VPN and try again. Is that correct?

Reply:
Yes that's correct

Reply:
They get a message at the top of the client that let's them know that the server location has become unavailable. This happens before they even deactivate. 

I added some log statements to capture what's happening